templates/web/fixmystreet.com/about/open311-api-info.html

```diff
@@ -0,0 +1,154 @@
+
+<dt>What is Open311?</dt>
+
+<dd>
+<p>In a nutshell, it's a free, international technology, known as an open standard, that allows civic reporting services and systems to 'talk to' each other. </p>
+<p>Setting up an Open311 API endpoint for FixMyStreet will enable you to receive reports from FixMyStreet.com and the app directly into your backend system(s) instead of via email. It will also allow you to close/update reports on FixMyStreet.com or the app when work is completed.</p>
+<p>Open311 isn’t new (we’ve been using it to connect to council systems since 2011), but it still isn't as widely known as it deserves to be. You can find out lots more at <a href="http://open311.org/">open311.org</a>.</p>
+
+</dd>
+
+<dt>How do I get started with Open311?</dt>
+
+<dd>
+<p>Setting up an Open311 endpoint yourself to provide access to one or more of your current services shouldn’t take you more than a few days of development: it usually only needs the support of your technical team and some project management time.</p>
+<p>We’re happy to give you some advice if you are just getting started, and you can <a href="https://github.com/mysociety/fixmystreet/wiki/Open311-FMS---Complete-Spec">find our full Open311 specification here.</a></p>
+
+<ul>
+  <li>First, let us know you are planning to set up an Open311 endpoint by emailing support@fixmystreet.com. This way we know to expect your API key and can also provide some advice where appropriate.</li>
+  <li>Find everything you need to know in the complete tech spec for the Open311 endpoint here: https://github.com/mysociety/fixmystreet/wiki/Open311-FMS---Complete-Spec</li>
+  <li>Once you've got a functional Open311 testing endpoint setup, email us the URL and API key and we’ll configure our FixMyStreet staging site to send appropriate FixMyStreet reports to that endpoint. From here you can inspect the data you receive and make adjustments until you’re happy.</li>
+  <li>You can then implement the spec we have given you access to, this includes</li>
+    <ul>
+      <li>Receiving reports (required)</li>
+      <li>Opening and closing reports (optional)</li>
+      <li>Listing services / Categories including extra questions (required)</li>
+      <li>Sending and receiving updates (optional)</li>
+      <li>Option to pull reports into the Open311 endpoint, this refers to reports not originally made in FixMyStreet </li>
+    </ul>
+  <li>Once you're satisfied it’s working and you have a fully compliant endpoint, email support@fixmystreet.com and we’ll update the live site to point at your live endpoint.</li>
+</ul>
+
+<p><strong>Please note:</strong> Once set up, the Open311 endpoint will override the list of categories visible on the FixMyStreet.com site and app according to those used within the connected backend system. Any existing FixMyStreet categories will be removed UNLESS you still want some additional categories to remain as email only (please let us know this beforehand).</p>
+
+</dd>
+
```

Comment:
I think you want to add a `</dl>` here to close this question/answer block, then make the `<dt>Frequently asked questions</dt>`  a `<h2>`